lua/lspconfig/configs.lua

```diff
@@ -33,26 +33,25 @@ function configs.__newindex(t, config_name, config_def)
   -- The config here is the one which will be instantiated for the new server,
   -- which is why this is a function, so that it can refer to the settings
   -- object on the server.
-  local function add_callbacks(config)
-    config.callbacks["window/logMessage"] = function(err, method, params, client_id)
+  local function add_handlers(config)
+    assert(not config.callbacks, "lsp.callbacks has been deprecated. See here for more: https://github.com/neovim/neovim/pull/12655")
```

Comment:
```suggestion
    assert(config.callbacks, "lsp.callbacks has been obsoleted. See here for more: https://github.com/neovim/neovim/pull/12655")
```

In some programming languages, "deprecated" may still be used, but it can mean that it will be removed in the future.
I think "absolute" is clearer than "deprecated".